packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -55,6 +58,21 @@ export function getPropGetters<
 
         if (isTargetWithinAutocomplete === false) {
           store.dispatch('blur', null);
+
+          // If requests are still running when the user closes the panel, they
+          // could reopen the panel once they resolve.
+          // We want to avoid any subsequent query and keep the panel closed.
```

Comment:
Mentioning why we don't want this behavior helps understand why it's important imo. Wdyt about mentioning something like "unsolicited UI behavior".

```suggestion
          // We want to prevent any subsequent query from reopening the panel because
          // it would result it an unsolicited UI behavior.
```